packages/nimble-blazor/NimbleBlazor/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ni/nimble-blazor",
+  "version": "16.0.1",
+  "description": "Blazor components for the NI Nimble Design System",
+  "scripts": {
+    "pack": "cross-env-shell dotnet pack -c Release -p:PackageVersion=$npm_package_version --output ../dist",
+    "invoke-publish": "npm run invoke-publish:clean-nuget && npm run pack && npm run invoke-publish:nuget && npm run invoke-publish:npm -- ",
+    "invoke-publish:clean-nuget": "rimraf --glob \"dist/NimbleBlazor.*.nupkg\"",
+    "invoke-publish:nuget": "cross-env-shell dotnet nuget push \"dist/NimbleBlazor.*.nupkg\" -k $NUGET_SECRET_TOKEN -s \"https://api.nuget.org/v3/index.json\"",
+    "invoke-publish:npm": "echo \"noop command to swallow npm args\""
```

Comment:
fyi, follow the pattern update here: https://github.com/ni/nimble/pull/2039/files#r1581350041